src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -86,7 +86,6 @@ public function __construct(BasketManagerInterface $basketManager, BasketElement
      *
      * @QueryParam(name="page", requirements="\d+", default="1", description="Page for baskets list pagination (1-indexed)")
      * @QueryParam(name="count", requirements="\d+", default="10", description="Number of baskets by page")
-     * @QueryParam(name="orderBy", array=true, requirements="ASC|DESC", nullable=true, strict=true, description="Query baskets basket by clause (key is field, value is direction")
      *
      * @View(serializerGroups="sonata_api_read", serializerEnableMaxDepthChecks=true)
```

Comment:
Can you change `"sonata_api_read"` by `{"sonata_api_read"}`?

So we fix all the issues at once? It should be simple. It should be also changed on the ApiDoc above

Reply:
I was going to ask about this - should I do it in this PR or in a separate one considering that `orderBy` is fixed differently in master?

Reply:
separate commit would be good enough

Reply:
👍